src/app/checkout/service.js

```diff
@@ -74,6 +74,18 @@ angular.module('checkoutModule')
                 return defer.promise;
             };
 
+            loadTokens = function () {
```

Comment:
please add comment and describe what tokens are and what is a difference between token and payment method (word token can mean a lot of things)